gnome-initial-setup/pages/location/gis-location-page.c

```diff
@@ -687,6 +687,25 @@ gis_location_page_constructed (GObject *object)
 
   update_timezone (page);
 
+  /*
+   * When running on a Live session, it's important to try to make the best
+   * usage of the system clock. Timedate1 has a handy method for that.
```

Comment:
rather than "best usage" and "handy method", can we describe what this call achieves?